packages/next/src/server/lib/cache-control.ts

```diff
@@ -8,27 +8,28 @@ import { CACHE_ONE_YEAR } from '../../lib/constants'
  * value for this option.
  */
 export type Revalidate = number | false
-export type ExpireTime = number
 
-export function formatRevalidate({
-  revalidate,
-  expireTime,
-}: {
+export interface CacheControl {
   revalidate: Revalidate
-  expireTime?: ExpireTime
-}): string {
+  expire?: number
```

Comment:
I'd strongly recommend instead to use:

```
expire: number | undefined
```

To ensure that every call-site that creates a new `CacheControl` will be forced to explicitly set it to `undefined` rather than the silent failure case now.